src/test/plugins/mvcc_delete_plugin_test.cpp

```diff
@@ -0,0 +1,153 @@
+#include <limits>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "../../plugins/mvcc_delete_plugin.hpp"
+#include "../utils/plugin_test_utils.hpp"
+#include "concurrency/transaction_manager.hpp"
+#include "expression/expression_functional.hpp"
+#include "expression/pqp_column_expression.hpp"
+#include "operators/get_table.hpp"
+#include "operators/projection.hpp"
+#include "operators/table_scan.hpp"
+#include "operators/update.hpp"
+#include "operators/validate.hpp"
+#include "storage/storage_manager.hpp"
+#include "storage/table.hpp"
+#include "utils/load_table.hpp"
+#include "utils/plugin_manager.hpp"
+
+namespace opossum {
+
+class MvccDeletePluginTest : public BaseTest {
```

Comment:
I just realzed that you are not testing the plugin as such. Could you add a test that simply runs the plugin and checks for the expected effects? You might want to put this into hyriseSystemTest and reduce the cycle times for the test.

Reply:
We are on it, but we encountered 3 instead of 2 references to the chunk which should be physically deleted next. Allowing 3 references does not seem to cause any problems but we still try to figure out where the additional reference comes from.